src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -281,4 +297,11 @@ public abstract partial class RegexRunnerFactory
         protected RegexRunnerFactory() { }
         protected internal abstract System.Text.RegularExpressions.RegexRunner CreateInstance();
     }
+    public ref struct SplitEnumerator
+    {
+        public ReadOnlySpan<char> Current { get; private set; }
+        public SplitEnumerator GetEnumerator() => this;
+        public ReadOnlySpan<char> Input { get; }
```

Comment:
Why is this Input property needed?

Reply:
Also, `{ get; }` should not be used in contracts; it should be `{ get { throw null; } }` or something similar; otherwise this is going to generate the backing field.